DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -210,5 +220,23 @@ public void UpdateDelegateUsers(string firstName, string surname, string email,
                 new { firstName, surname, email, profileImage, ids }
             );
         }
+
+        public void UpdateDelegateUserCentrePrompts(int id, int jobGroupId, string? answer1, string? answer2, string? answer3,
+            string? answer4, string? answer5, string? answer6)
```

Comment:
Could we put every param on a new line if we need to break the line up to make it really easy to scan over all of them?